### PR TITLE
Universal/fix gles max visible lights

### PR DESCRIPTION
**Pull request by hkr**
### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [x] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [ ]  Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [x] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Review by phi-lira — APPROVED** [review]
Nice. So we have a SHADER_API_GLES30 now! 
 Do we also have one specific for 3.1+? I guess GLES3 and not GLES30?

 This PR depends on a trunk PR for the new define right?

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added the Detail Inputs setting group in the Lit shader (Lit.shader).
 - Added Smooth shadow fading.
 
+### Changed
+- The maximum number of visible lights when using OpenGL ES 3.x on Android now depends on the minimum OpenGL ES 3.x version as configured in PlayerSettings.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/CHANGELOG.md:18]: We should probably state the limits here to be more clear to user when reading the changelog.

**Comment by ellioman** [comment]
Just adding the reminder here that we need to bump the minimum Unity Version for all Graphics packages, in this PR, once the backport to 2020.2 lands.

**Review by ellioman — APPROVED** [review]

**Comment by hkr** [comment]
> Nice. So we have a SHADER_API_GLES30 now!
> Do we also have one specific for 3.1+? I guess GLES3 and not GLES30?

`SHADER_API_GLES30` is "platform caps keyword" (the other SHADER_API_* are not), so I did *not* add `SHADER_API_31`, etc because it has some cost. Could be done if needed. 

> 
> This PR depends on a trunk PR for the new define right?

yes

**Comment by ellioman** [comment]
The errors on DX11 and Stereo are known issues and not caused by this PR

**Review by alindmanUnity — APPROVED** [review]

**Pull request merged**